core/domain/suggestion_registry.py

```diff
@@ -1095,3 +1095,109 @@ def reject(self, reviewer_id, rejection_message):
         SuggestionTranslateContent),
     suggestion_models.SUGGESTION_TYPE_ADD_QUESTION: SuggestionAddQuestion
 }
+
+
+class CommunityContributionStats(python_utils.OBJECT):
+    """Domain object for the CommunityContributionStatsModel.
+
+    Attributes:
+        translation_reviewer_counts_by_lang_code: dict. A dictionary where the
+            keys represent the language codes that translation suggestions are
+            offered in and the values correspond to the total number of
+            reviewers who have permission to review translation suggestions in
+            that language.
+        translation_suggestion_counts_by_lang_code: dict. A dictionary where
+            the keys represent the language codes that translation suggestions
+            are offered in and the values correspond to the total number of
+            translation suggestions that are currently in review in that
+            language.
+        question_reviewer_count: int. The total number of reviewers who have
+            permission to review question suggestions.
+        question_suggestion_count: int. The total number of question
+            suggestions that are currently in review.
+    """
+
+    def __init__(
+            self, translation_reviewer_counts_by_lang_code,
+            translation_suggestion_counts_by_lang_code,
+            question_reviewer_count, question_suggestion_count):
+        self.translation_reviewer_counts_by_lang_code = (
+            translation_reviewer_counts_by_lang_code
+        )
+        self.translation_suggestion_counts_by_lang_code = (
+            translation_suggestion_counts_by_lang_code
+        )
+        self.question_reviewer_count = question_reviewer_count
+        self.question_suggestion_count = question_suggestion_count
+
+    def validate(self):
+        """Validates the CommunityContributionStats object.
+
+        Raises:
+            ValidationError. One or more attributes of the
+                CommunityContributionStats object is invalid.
+        """
+        for language_code, reviewer_count in (
+                self.translation_reviewer_counts_by_lang_code.items()):
+            if reviewer_count < 0:
```

Comment:
Do we need to check the type of the count?

Reply:
I put a check in to make sure that the counts are ints in the follow up PR.